packages/why/why.2.40/opam

```diff
@@ -0,0 +1,42 @@
+opam-version: "1.2"
+maintainer: "Claude.Marche@inria.fr"
+authors: [
+  "Jean-Christophe Filliâtre"
+  "Claude Marché"
+  "Yannick Moy"
+  "Romain Bardou"
+]
+homepage: "http://krakatoa.lri.fr/"
+doc: ["http://krakatoa.lri.fr/"]
+bug-reports: "https://gforge.inria.fr/tracker/?atid=4012&group_id=999&func=browse"
+dev-repo: "https://scm.gforge.inria.fr/anonscm/git/why/why.git"
+license: "LGPL-2.1"
+tags: [
+  "deductive"
+  "program verification"
+  "specification"
+  "automated theorem prover"
+  "interactive theorem prover"
+  "Java"
+  "JML"
+  "C"
+  "ACSL"
+]
+available: [ ocaml-version >= "4.02.0" ]
+build: [
+  ["./configure" "--prefix" prefix]
+  [make]
+]
+install: [
+  [make "install"]
+]
+remove: [
+  ["rm" "%{prefix}%/bin/jessie"]
+  ["rm" "%{prefix}%/bin/krakatoa"]
+  ["rm" "-rf" "%{prefix}%/lib/why"]
+]
+depends: [
+  "why3" {>= "0.86.1" & <= "0.88.3" }
+  "frama-c" {= "20171101"}
+  "conf-autoconf"
```

Comment:
Could you flag this dependency with ```{build}``` please ? I doubt autoconf is used outside of the build phase and thus it is safe to assume that there is no runtime dependency on it.